src/riak_kv_compile_tab.erl

```diff
@@ -165,6 +166,16 @@ get_ddl(BucketType) when is_binary(BucketType) ->
             notfound
     end.
 
+%%
+-spec get_all_compiled_ddls() -> term() | notfound.
+get_all_compiled_ddls() ->
+    case dets:match(?TABLE, {'_','_','$1','_','compiled'}) of
```

Comment:
Can more than one version of a table be active at the same time on the same node?